include/itkANTSRegistration.h

```diff
@@ -108,14 +108,22 @@ class ANTSRegistration : public ProcessObject
   itkSetStringMacro(TypeOfTransform);
   itkGetStringMacro(TypeOfTransform);
 
-  /** Set/Get the initial fixed transform. */
+  /** Set/Get the initial transform. */
```

Comment:
can we still describe the direction of this transform, e.g. "forward, fixed to moving" (if that is what it is)?